### ISSUE TITLE
:building_construction: `help wanted`

### ISSUE THREAD

**Issue by AroKol78** [gorhill/uBlock-for-firefox-legacy#329]
will there be anyone with permission to put together even simple things - #319, #328

**Comment by hawkeye116477** [comment]
Well, unfortunately after gorhill gave up on this, JustOff was only person who received permissions and from description on his profile results that he's probably from Ukraine, so he has more to worry about currently.

**Comment by AroKol78** [comment]
yeah i know JustOff is probably on war now and ... i hope he comes back alive and well.
i thought gorhill would give someone permission in an extraordinary situation
and if not, someone will make a fork.

(tak wiem, że JustOff jest teraz prawdopodobnie na wojnie i ... mam nadzieję, że wróci cały i zdrowy.
myślałem, że gorhill udzieli komuś uprawnień w nadzwyczajnej sytuacji.
a jak nie to może ktoś zrobi widelec.)

**Comment by FrostKnight** [comment]
I wonder if justoff has the power to give permission to other people to help him.

Seems thing have calmed down more on UXP lately...

**Comment by THEtomaso** [comment]
@hawkeye116477:

> JustOff was only person who received permissions

Would you be interested?
If so, we could ping gorhill about it.

**Comment by Sa-Ja-Di** [comment]
> Well, unfortunately after gorhill gave up on this, JustOff was only person who received permissions and from description on his profile results that he's probably from Ukraine, so he has more to worry about currently.

People can fork this project, rebrand it and can continue legally as a fork, that is no issue

**Comment by garoto** [comment]
But forking this repo isn't a solution either. If you didn't noticed, this repo is owned by gorhill himself, which is a huge deal by any and all metrics.

**Comment by FrostKnight** [comment]
> yeah i know JustOff is probably on war now and ... i hope he comes back alive and well. i thought gorhill would give someone permission in an extraordinary situation and if not, someone will make a fork.
> 
> (tak wiem, że JustOff jest teraz prawdopodobnie na wojnie i ... mam nadzieję, że wróci cały i zdrowy. myślałem, że gorhill udzieli komuś uprawnień w nadzwyczajnej sytuacji. a jak nie to może ktoś zrobi widelec.)

Actually, about that... I think there is a better solution...

https://github.com/dhowe/AdNauseam/releases/tag/v3.7.802

This is the last version I mentioned above
Btw, I will probably back it up just in case.


Which, should be something obvious already...

;)

But yeah, if ublock origin is in this current state, this alas, might be the better option.

I don't know if @dhowe would be willing to have a link for this, if someone was willing to step up. If he would allow someone to work on it, on his direct repo, or just link it from somewhere else.

Regardless, if he was willing, he would want to have it say:

This is not directly supported by us and if there are issues, you must contact the maintainer of this version.

You would also need to mention, to use this, you cannot have the blocklist on. 

@garoto 
@AroKol78 
@hawkeye116477 
@tailHey 
@Sa-Ja-Di 

This does include @dhowe 

I understand though if he has reservations about supporting this idea directly though.

Anywho let me know if any of you like this idea...

**Comment by dhowe** [comment]
greetings all - what exactly is being suggested re: adnauseam ?

**Comment by FrostKnight** [comment]
> greetings all - what exactly is being suggested re: adnauseam ?

I was suggesting, that the people who want to continue a legacy firefox/uxp adblocker, instead try to do so from your last legacy firefox version...
And you could have the legacy version continued, on your repo, or a link to a different person maintaining it, if someone has interest in continuing an actual good adblocker for uxp based web browsers.

Also, you would leave a note saying, to cite issues with the new maintainer, not you.
And that you need to disable the blocklist to use said addon.

I made mention of this, because gorhill and justoff, seem to be unavailable, thus this repository is basically dead in a nutshell till someone else gets permission.

It is better to have an adblocker that works as is actively maintained than one that is barely alive.

Regarding uxp.

Btw, its not even just palemoon, there are other forks of the xul unified project besides that one.

I hope this clarifies what I meant. 

:)

**Comment by dhowe** [comment]
happy to place a link if its useful...

**Comment by FrostKnight** [comment]
> happy to place a link if its useful...

If that's the case, all that is needed, is for one of these contributors/people above to see the recent comments,
@THEtomaso 
@garoto 
@hawkeye116477 

Either way,  just need to see if any of these people have interest.

;)  

I am pondering emailing one of them 
Thanks for the offer though @dhowe 

Only thing is, just gotta see if anyone will respond or if they gave up.

Hopefully they will respond. :)

**Comment by FrostKnight** [comment]
Would be nice if someone could take over ublock origin legacy other than justoff, till he returns.

Which, who knows when or if...

:(
